.rubocop.yml

```diff
@@ -3,6 +3,8 @@ AllCops:
     - mixlib-install.gemspec
 Style/AlignParameters:
   EnforcedStyle: with_first_parameter
+Style/BlockDelimiters:
```

Comment:
curious. this project uses chef-style so the rubocop config should be removed altogether.